gallery/src/styles/NxViewportSized/NxViewportSizedExample.tsx

```diff
@@ -0,0 +1,624 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxTable, NxTableHead, NxTableRow, NxTableCell, NxTableBody, NxBackButton, NxInfoAlert }
+  from '@sonatype/react-shared-components';
+
+export default function NxViewportSizedScrollableExample() {
+  return (
+    <main className="nx-page-main nx-viewport-sized">
+      <NxBackButton targetPageTitle="nx-viewport-sized Docs" href="#/pages/nx-viewport-sized"/>
+      <section className="nx-tile nx-viewport-sized__container">
+        <header className="nx-tile-header">
+          <div className="nx-tile-header__title">
+            <h2 className="nx-h2">A table, with some stuff above</h2>
+          </div>
+        </header>
+        <div className="nx-tile-content nx-viewport-sized__container">
+          <p className="nx-p">
+            Observe that the table below, along with its containing tile, shrinks to fit the viewport and gets a
+            scrollbar. As the page size is adjusted, the table size changes accordingly.
+          </p>
+          <p className="nx-p" style={{ width: '400px' }}>
+            This paragraph has an explicit, non-100% width. It is still left-aligned within the tile just like it
+            would be in block layout.
+          </p>
+          <p className="nx-p">
+            Single line paragraph.
+          </p>
+          <NxInfoAlert>
+            This alert is stretched as expected.
```

Comment:
I don't understand this sentence, the alert is not stretched in any way that I can see. It appears correctly.